dxvk.conf

```diff
@@ -707,18 +707,3 @@
 #
 # Supported values:
 # - True/False
-
-# d3d9.countLosableResources = True
```

Comment:
This line shouldn't be deleted

Reply:
Good point, guess I was a bit overzealous there :smiling_face_with_tear: 

Fixed.